src/functions/UserData.js

```diff
@@ -38,13 +38,30 @@ export async function getUserDatabase() {
   }
 }
 
+// gets challenger's data from firebase
+export async function getChallengerDatabase() {
+  const userData = await getUserDatabase();
+  let uid = userData.invitedBy;
+  let docRef = doc(db, "users", uid);
+  let docSnap = await getDoc(docRef);
+  if (!docSnap.exists()) {
+    throw new Error("challenger doc does not exist");
+  } else {
+    console.log("challenger db", docSnap.data());
+    return docSnap.data();
+  }
+}
+
 // call this function when user completes an action
 // ADD email user after certain completed actions
 export async function completedAction(action) {
   const userData = await getUserDatabase();
   const uid = auth.getAuth().currentUser.uid;
   // sign up for election reminders
   if (action === "election reminders") {
+    if (!userData.startedChallenge) {
```

Comment:
I assume this is saying that `if it is a player`? 
Should we still send the email to the player who is completing this action, but has also started a challenge (thus a hybrid-user)?

Reply:
Yeah, I talked to Tyler about this; I will be updating this to also send emails to challengers. Tyler had to update the email's content to exclude the reference to a challenger getting a badge specifically when sending this email to a challenger.